public/apps/account/utils.tsx

```diff
@@ -31,7 +31,9 @@ export async function fetchAccountInfoSafe(http: HttpStart): Promise<AccountInfo
 export async function logout(http: HttpStart, logoutUrl?: string): Promise<void> {
   await httpPost(http, API_AUTH_LOGOUT);
   setShouldShowTenantPopup(null);
-  window.location.href = logoutUrl || http.basePath.serverBasePath;
+  const nextUrl = encodeURIComponent(http.basePath.serverBasePath);
+  window.location.href =
+    logoutUrl || `${http.basePath.serverBasePath}/app/login?nextUrl=${nextUrl}`;
```

Comment:
This doesn't work after I cleared out browser cache

Please verify this change can correctly redirect URL after logged out

Reply:
can you specify the url it is redirecting to?

Reply:
it redirects to `localhost:5601/app/login?nextURL=`

where `nextURL` is empty and thus not being able to login to home page

<img width="1593" alt="Screen Shot 2021-06-07 at 9 31 25 AM" src="https://user-images.githubusercontent.com/79167042/121056582-31774a80-c773-11eb-83b4-8d52ad6c84be.png">

Reply:
got it, thanks for pointing out, this is indeed a bug. Actually I came to know that we need to test for two flows: 1. `yarn start --oss` 2. `yarn start --oss --no-base-path`. When there is no basepath(2 option) then the nextUrl will be empty and the /app/login page will keep on looping

Reply:
will raise a PR to address this issue

Reply:
Please fix this on main branch before proceeding

Reply:
this [PR](https://github.com/opensearch-project/security-dashboards-plugin/pull/778/files) fixes that.